spec/immutable_struct_spec.rb

```diff
@@ -103,14 +103,28 @@ def self.from_array(array)
   end
 
   describe "to_h" do
-    it "should include the output of params and block methods in the hash" do
+    it "should not include the output of params and block methods in the hash" do
```

Comment:
Per my comments on the issue, I'm not in favor of this, at least as the default.  This is a breaking change and we wouldn't be able to merge it without a lot of testing on our end.